findMsBuild.cmd

```diff
@@ -0,0 +1,27 @@
+@echo off
+
+IF NOT DEFINED VSVERSION SET VSVERSION=15.0
```

Comment:
Will VSVERSION be defined in MicroBuild as 14.0 if we run in the wrong pool?  And does that matter since we presumably can't build anyway in that case?